ionic-angular-template/src/services/auth.service.ts

```diff
@@ -75,7 +78,7 @@ export class AuthService {
       const currentUser = await firebase.auth().signInWithEmailAndPassword(email, newPassword);
       const wallet = ethers.Wallet.createRandom();
       const encryptPromise = await wallet.encrypt(newPassword);
-      this.api.put(`${environment.apiUrl}/users/wallet`, {uid: currentUser.user.uid, wallet: encryptPromise}).subscribe();
+      await this.api.put(`${environment.apiUrl}/users/wallet`, {uid: currentUser.user.uid, wallet: encryptPromise});
```

Comment:
change the name of the variable `encryptPromise`. This is not encryptPromise. This is encrypted json file.